pennylane/fermi/fermionic.py

```diff
@@ -511,6 +511,43 @@ def from_string(fermi_string):
     return FermiWord({(i, int(s[:-1])): s[-1] for i, s in enumerate(operators)})
 
 
+def _to_string(fermi_op, of=False):
+    r"""Return a string representation of the :class:`~.FermiWord` object.
+
+    Args:
+        fermi_op (FermiWord): the fermionic operator
+        of (bool): whether to return a string representation in the same style as OpenFermion using
+                    the shorthand: 'q^' = a^\dagger_q 'q' = a_q. Each operator in the word is
+                    represented by the number of the wire it operates on
+
+    >>> w = FermiWord({(0, 0) : '+', (1, 1) : '-'})
```

Comment:
This is a private function so this is not required, but typically we place example code in a separate section 
```suggestion

    *Example:*
    
    .. code:: pycon
    
        >>> w = FermiWord({(0, 0) : '+', (1, 1) : '-'})
```

Reply:
Can be left as is for now,